### ISSUE TITLE
Error on typescript definition for DynamoDB.DocumentClient constructor

### ISSUE THREAD

**Issue by gozup** [aws/aws-sdk-js#1499]
Hi,

I wanted to notice that the TS definition file for DynamoDB.DocumentClient() is not correct.
When I instantiate this object with some options, options are not recognized. For instance:

```javascript
let client: DynamoDB.DocumentClient = new DynamoDB.DocumentClient({ region: "us-east-1"});
```

Then, TypeScript compiler (2.3.2) returns:

> Error:(16, 5) TS2345:Argument of type '{ region: string; }' is not assignable to parameter of type 'DocumentClientOptions | undefined'.
>   Object literal may only specify known properties, and 'region' does not exist in type 'DocumentClientOptions | undefined'.

And indeed, after checking,  in the declaration file the constructor of DynamoDB.DocumentClient should receive a `DocumentClient.DocumentClientOptions` interface that must contain a `params` key that embed the options. Ex:

```javascript
let client: DynamoDB.DocumentClient = new DynamoDB.DocumentClient({ params: { region: "us-east-1"}));
```

This way, TypeScript is happy, but the option is actually not set for the object instance. So it throws an error :(

Hope you can fix it quickly.
Cheers!

**Comment by gozup** [comment]
Sorry, it's fixed in the last release...
My bad

**Comment by gozup** [comment]
Event fix the last version (2.48.0)
